### PR TITLE
[Common] data size of extra tensor

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Fix util function, getting tensor size of extra tensor.

### CODE REVIEW

File: gst/nnstreamer/nnstreamer_plugin_api_util_impl.c [hunk 1]
```diff
@@ -422,17 +422,21 @@ gst_tensors_info_free (GstTensorsInfo * info)
...
 gst_tensors_info_get_size (const GstTensorsInfo * info, gint index)
 {
+  GstTensorInfo *_info;
   gsize data_size = 0;
   guint i;
 
   g_return_val_if_fail (info != NULL, 0);
   g_return_val_if_fail (index < (gint) info->num_tensors, 0);
 
   if (index < 0) {
```
Comment (songgot) [inline gst/nnstreamer/nnstreamer_plugin_api_util_impl.c:432]: How about fixing it with ```if (index == -1)```?
The documentation only mentions -1. (* @param index the index of tensor (-1 to get total size of tensors))

Comment (jaeyun-jung) [reply gst/nnstreamer/nnstreamer_plugin_api_util_impl.c:432]: supposed negative index value to get total tensor size in the past.

Comment (songgot) [reply gst/nnstreamer/nnstreamer_plugin_api_util_impl.c:432]: Okay, Thank you.

**Review by songgot — APPROVED** [review]
LGTM

**Review by anyj0527 — APPROVED** [review]
LGTM, Thanks!

**Review by niley7464 — APPROVED** [review]
LGTM :)

**Pull request merged**